source/index.html.haml

```diff
@@ -19,10 +19,14 @@ layout: home
 
   %h2 CMO is back for 2018!
 
-  = link_to("/apply", class: "button") do
-    = content_tag :button, "Apply Now", class: "alpha-button"
+  = link_to("#call-to-men", class: "button scroll-on-page-link") do
+    = content_tag :button, "Learn More", class: "alpha-button"
 
-  %a.scroll.scroll-on-page-link{href: "#call-to-men"}
+  = link_to("https://euroteamoutreach.org/blog/2018/07/go-forward/",
```

Comment:
Obviously not ideal to hard-code this link, but for now it's a quick fix. In future we should probably set up some kind of `/current` route that would intelligently redirect to the latest eReport.

Reply:
Except that the latest eReport is not always going to be CMO-related.

Reply:
Right. My thought is that once we introduce tagging on the ETO blog (which we've not yet done) we can use that to always point to the latest post that's tagged "ereport". Alternatively, we could just remove the button during the off-season.